src/aws_credentials.pas

```diff
@@ -19,13 +19,46 @@ interface
   //rtl
   sysutils,
   classes,
+  dateutils,
   //synapse
   synacode,
   synautil,
+  synacrypt,
+  //terceiros
+  SynCrypto,
   //aws
   aws_http;
 
+
 type
+
+  IAWSSignatureHMAC256 = interface(IInterface)
+  ['{9158D9A2-7ABA-4126-9F63-264E947AC60A}']
+    function AccessKey: string;
+    function DataStamp: string;
+    function RegionName: string;
+    function ServiceName: string;
+    function Signature: TSHA256Digest;
```

Comment:
@almp1 why do we need this new type `TSHA256Digets`? Did you search if Synapse there is some similar?